libs/langchain/langchain/llms/vertexai.py

```diff
@@ -449,7 +455,7 @@ def _generate(
         generations: List[List[Generation]] = []
         for result in response.predictions:
             generations.append(
-                [Generation(text=prediction[self.result_arg]) for prediction in result]
+                [Generation(text=result)]
```

Comment:
It looks like this a change on Vertex side, so probably it's a good idea to support both response formats.

Otherwise we might break things for [previously deployed and] working endpoints.

Reply:
yes and no. I think we should follow the principle of the underlying SKD i.e. Vertex AI SDK. it seems that it was a breaking change decided by Google so I would say langchain should only reflect that